src/asmcnc/comms/router_machine.py

```diff
@@ -487,7 +487,7 @@ def write_z_head_laser_offset_values(self, enabled, X, Y):
             file.close()
             self.laser_offset_x_value = float(X)
             self.laser_offset_y_value = float(Y)
-            if enabled == "True" or enabled == True: self.is_laser_enabled = True
```

Comment:
These should be refactored to not be checking against a string

Reply:
It looks to be a pretty big refactoring job, I think it should be its own separate ticket because it would also require some testing to make sure everything still works like before

Reply:
Then you need to put == true back, (or remove it entirely as it will never be a book when reading from a text file). As the string "false" is True.

Reply:
Ah I was not aware of that, updated